src/goto-instrument/contracts/contracts.cpp

```diff
@@ -772,35 +769,39 @@ void code_contractst::apply_function_contract(
   is_fresh.add_memory_map_decl(new_program);
 
   // Insert assertion of the precondition immediately before the call site.
-  exprt::operandst requires_conjuncts;
-  for(const auto &r : type.requires())
+  for(const auto &clause : type.requires())
   {
-    requires_conjuncts.push_back(
-      to_lambda_expr(r).application(instantiation_values));
-  }
-  auto requires = conjunction(requires_conjuncts);
-  requires.add_source_location() =
-    requires_conjuncts.empty() ? type.source_location()
-                               : type.requires().front().source_location();
-  if(!requires.is_true())
-  {
-    if(has_subexpr(requires, ID_exists) || has_subexpr(requires, ID_forall))
-      add_quantified_variable(requires, mode);
-
-    goto_programt assertion;
-    converter.goto_convert(code_assertt(requires), assertion, mode);
-    assertion.instructions.back().source_location_nonconst() =
-      requires.source_location();
-    assertion.instructions.back().source_location_nonconst().set_comment(
-      "Check requires clause");
-    assertion.instructions.back().source_location_nonconst().set_property_class(
-      ID_precondition);
-    is_fresh.update_requires(assertion);
-    new_program.destructive_append(assertion);
+    auto instantiated_clause =
+      to_lambda_expr(clause).application(instantiation_values);
+    if(!instantiated_clause.is_true())
+    {
+      if(
+        has_subexpr(instantiated_clause, ID_exists) ||
```

Comment:
This formatting/indentation is weird.

Reply:
All the formatting/indentation is done by `clang-format`.